docs/source/readme.rst

```diff
@@ -1,64 +1,115 @@
 POT: Python Optimal Transport
 =============================
 
-|PyPI version| |Anaconda Cloud| |Build Status| |Codecov Status|
-|Downloads| |Anaconda downloads| |License|
+|PyPI version| |Anaconda Cloud| |Build Status| |Build Status| |Codecov
+Status| |Downloads| |Anaconda downloads| |License|
 
 This open source Python library provide several solvers for optimization
 problems related to Optimal Transport for signal, image processing and
 machine learning.
 
 Website and documentation: https://PythonOT.github.io/
 
-POT provides the following solvers:
-
--  OT Network Flow solver for the linear program/ Earth Movers Distance
-   [1].
--  Entropic regularization OT solver with Sinkhorn Knopp Algorithm [2],
-   stabilized version [9][10] and greedy Sinkhorn [22] with optional GPU
-   implementation (requires cupy).
+Source Code (MIT): https://github.com/PythonOT/POT
+
+POT provides the following generic OT solvers (links to examples):
+
+-  `OT Network Simplex
+   solver <https://pythonot.github.io/auto_examples/plot_OT_1D.html>`__
+   for the linear program/ Earth Movers Distance [1] .
+-  `Conditional
+   gradient <https://pythonot.github.io/auto_examples/plot_optim_OTreg.html>`__
```

Comment:
within sphinx I would not hard code the links but rely on cross refs within sphinx files. Otherwise the old version of the doc will keep linking to new files

Reply:
OK done,

plus if you type make rdoc the readme.md is converted to readme.rst in the doc with relative path (sed magic).